src/pages/player-info.md

```diff
@@ -0,0 +1,98 @@
+
+![](/images/discord-overlay-example.jpg)
+
+Once you join Discord, send a note anyone listed under Admins in the Discord (Matty, Jeremy, or Dan) and let us know you are in; we will grant you the proper role to give you access to the “Game” voice channel (and the “players” text channel). The “Game” voice channel is what you will need to join/connect audio to during the actual stream/event. The text channel is just in case we want it. 
+
+You also should set your “nickname” on our Discord server to your actual name - that is what will display in the overlay instead of your Discord username. It will just look nicer :)
+
+Note that the Discord is public for members of the community; there are private channels just for players and moderators though!
+
+## Schedule
+Episodes are streamed at **20:00 UTC+1** (EMEA league) or **20:00 UTC-6** (Americas league) on a **Tuesday**!  The game itself (all rounds) will likely take about an hour, but we will schedule an extra 30 minutes prior for “green room” time to make sure everything is working as expected. Please join the Discord voice channel (“Game”) **30 minutes before show time on game night** so we can do tech checks, etc.
+
+You will receive a calendar invite for the game, which is the time you should join the "Game" channel in Discord and be ready to go.
+
+We will also provide the ability for a “practice” session (optional) which will be “classic” Drawful and Quiplash, which could be helpful for players who are new to these games. You will get a calendar invite for this practice game; it’s totally optional, but join if you want! It should be fun.
+
+Here is a rough schedule (all times are the time zone of the game)
+
+19:30: Players join Discord and do audio checks/tech checks, etc<br />
+19:50: Stream on twitch.tv/mattstratton starts with countdown to show timer<br />
+20:00: Show starts streaming live, audio from players is “hot”, etc. Hosts introduce the game and introduce the players<br />
+20:05: Start playing first game<br />
+20:30: Start playing second game<br />
+20:55: End of show; thank everyone for playing/watching, etc. 
+
+Note that these times are estimated; we aren’t 100% sure how long each round will take, etc, so it’s POSSIBLE the show might go a few minutes over.
+
+## FAQ
+*When will this happen?*<br />
+**EMEA League** - 
+Episodes stream at 20:00 UTC+1 on the second Tuesday of each month.<br />
+**Americas League** Episodes stream at 20:00 UTC-6 on the last Tuesday of each month.
```

Comment:
Those UTC markers are valid for standard time only; summer time shifts them both by +1.

```suggestion
**EMEA League** Episodes stream at 20:00 CET / Paris on the second Tuesday of each month.<br />
**Americas League** Episodes stream at 20:00 CT / Chicago on the last Tuesday of each month.
```